packages/functional-tests/tests/misc/fourOhFour.spec.ts

```diff
@@ -10,10 +10,10 @@ test.describe('404', () => {
     pages: { login, fourOhFour },
   }) => {
     await fourOhFour.goto('load');
-    expect(await fourOhFour.header.isVisible()).toBeTruthy();
-    expect(await fourOhFour.homeLink.isVisible()).toBeTruthy();
+    await expect(fourOhFour.header).toBeVisible();
+    await expect(fourOhFour.homeLink).toBeVisible();
```

Comment:
Note: All these were auto-fixes. I believe for the [`playwright/prefer-web-first-assertions`](https://github.com/playwright-community/eslint-plugin-playwright/blob/main/docs/rules/prefer-web-first-assertions.md) rule.

Reply:
We have used the incorrect code mentioned in [prefer-web-first-assertions](https://github.com/playwright-community/eslint-plugin-playwright/blob/main/docs/rules/prefer-web-first-assertions.md) in a lot of places. Should we create a separate ticket altogether to fix it?